dev-tools/mage/build.go

```diff
@@ -166,17 +166,6 @@ func GolangCrossBuild(params BuildArgs) error {
 			"only be executed within the golang-crossbuild docker environment")
 	}
 
-	defer DockerChown(filepath.Join(params.OutputDir, params.Name+binaryExtension(GOOS)))
-	defer DockerChown(filepath.Join(params.OutputDir))
-
-	mountPoint, err := ElasticBeatsDir()
-	if err != nil {
-		return err
-	}
-	if err := sh.Run("git", "config", "--global", "--add", "safe.directory", mountPoint); err != nil {
-		return err
-	}
-
```

Comment:
All of this was necessary because files written in the container were owner by root. They're now owned by the host user to begin with.

Reply:
I think that this relates to https://github.com/elastic/elastic-agent/issues/9433 cc @dliappis @fr4nc1sc0-r4m0n